templates/components/carousel.html

```diff
@@ -62,8 +62,9 @@
     {{/each}}
     {{#and arrows (if carousel.slides.length '>' 1)}}
     <button aria-label="{{lang 'carousel.arrowAriaLabel'}} {{carousel.slides.length}}" class="js-hero-next-arrow slick-next slick-arrow"></button>
+    {{> components/carousel-content-announcement slides_length=carousel.slides.length}}
     {{/and}}
-    {{#if play_pause_button}}
+    {{#and play_pause_button (if carousel.slides.length '>' 1)}}
```

Comment:
Could wrap it into parent (if carousel.slides.length '>' 1) if

Reply:
done